vcd/writer.py

```diff
@@ -464,6 +466,29 @@ def format_value(self, value, check=True):
         else:
             return '0' + self.ident
 
+class StringVariable(Variable):
+    """String variable as known by GTKWave.
+
+    Any "string" (character-chain) can be displayed as a change. This type is
+    only supported by GTKWave.
+
+    """
+
+    __slots__ = ()
+
+    def format_value(self, value, check=True):
+        """Format scalar value change for VCD stream.
+
+        :param value: a string, str()
+        :type value: str
+        :raises ValueError: for invalid *value*.
+        :returns: string representing value change for use in a VCD stream.
+
+        """
+        if check and (not isinstance(value, six.string_types) or ' ' in value):
+            raise ValueError('StringVariables can only accept non-space-containing-string-values. ({})'.format(value))
```

Comment:
This line is being flagged by `flake8` as being too long.

Also, the error message is different than in the `ValueError`'s raised by the other `Variable` subclasses.

I suggest something simpler/shorter like `'Invalid string value "{}"'.format(value)`. It's probably best to say 'string' instead of 'StringVariable' since end-users do not construct `StringVariable` instances directly.